### PR TITLE
Additional fixes for honister

### PR DESCRIPTION
**Pull request by ricardosalveti**
Additional fixes (mostly removing stuff already merged in the respective layers) as I review the base layer changes in lmp.

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by caiotpereira — APPROVED** [review]

**Review by mike-sul — COMMENTED** [review]
Why are we removing `python3-docker-compose_1.29.2.bb`? Is there some another recipe for `docker-compose`?

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend [hunk 1]
```diff
@@ -17,7 +17,6 @@ SRC_URI:append:libc-musl = " \
     file://utils.c-disable-tilde-as-it-is-not-supported-by-musl.patch \
 "
 
-PACKAGECONFIG[sota-tools] = "-DBUILD_SOTA_TOOLS=ON ${GARAGE_SIGN_OPS},-DBUILD_SOTA_TOOLS=OFF,glib-2.0 ostree,"
```
Comment (mike-sul) [inline meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:20]: How will `garage-sign` be downloaded if `sota-tools` are removed?

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend [hunk 2]
```diff
@@ -65,10 +64,6 @@ FILES:${PN}-lite = " \
                     "
 FILES:${PN}-lite-lib = "${nonarch_libdir}/lib${PN}_lite.so"
 FILES:${PN}-lite-dev = "${includedir}/${PN}-lite"
-FILES:${PN}-dev = " \
```
Comment (mike-sul) [inline meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:68]: I suppose it's needed for a custom client.

**Comment by ricardosalveti** [comment]
> Why are we removing `python3-docker-compose_1.29.2.bb`? Is there some another recipe for `docker-compose`?

Removed because the same recipe/version is provided by meta-virtualization now.

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:20]: This same line/content is now available in the aktualizr recipe from meta-updater, so nothing changes.

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend (hunk 2 above)
Comment (ricardosalveti) [reply meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:68]: Same here.

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend (hunk 1 above)
Comment (mike-sul) [reply meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:20]: btw, we can remove `sota-tools`/native building of akltualizr at all. Effectively, what we need is just `garage-sign` which is downloaded by the python script while aktualizr is being built with `-DBUILD_SOTA_TOOLS=ON ` option. The other binaries built with this option are either replaced by fiopush/fiosync or non needed at all (`garage-deploy`).

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Review by angolini — APPROVED** [review]

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:20]: Indeed, we can follow up with that cleanup then.

**Pull request merged**